docs/eventing/sources/pingsource.md

```diff
@@ -1,196 +1,261 @@
 ---
-title: "PingSource"
+title: "PingSource example"
 linkTitle: "PingSource"
-weight: 31
+weight: 10
 type: "docs"
+aliases:
+  - ../cronjob-source
 ---
 
-![version](https://img.shields.io/badge/API_Version-v1beta2-red?style=flat-square)
+This topic explains event-driven architecture in terms of producers and consumers and, to that end,
+shows how to configure PingSource as an event source targeting a Knative Service.
 
-A PingSource produces events with a fixed payload on a specified cron schedule.
+## Before you begin
 
-## Installation
+1. Set up [Knative Serving](../../../serving).
+1. Set up [Knative Eventing](../../../eventing).
 
-The PingSource source type is enabled by default when you install Knative Eventing.
+## Overview
 
-## Example
+In event-driven architecture the most straightforward use case is a producer creating an event and a
+consumer acting on that event.
 
-This example shows how to send an event every minute to a Event Display Service.
+### Producers
 
-### Creating a namespace
+Producers create events. Using GitHub as an example, the producer is the GitHub webhook that sends
+information when a pull request is created.
+Essentially, you ask the producer for events that they are capable of sending and you tell the
+producer where to send those events.
 
-Create a new namespace called `pingsource-example` by entering the following
-command:
+Producers send the event data per specifications that make it easy for consumers to handle it.
+Knative uses the CloudEvents specifications by default.
+For more information, see the [Cloud Events](https://cloudevents.io/) website.
 
-```shell
-kubectl create namespace pingsource-example
-```
+### Consumers
```

Comment:
same as above, seems like generic info for an "Event sources" landing page

Reply:
Same as above, @omerbensaadon asked for this doc to be better furnished with detail as part of an effort to making getting started with Eventing easier.

Reply:
@abrennan89, it's good content regardless. Where would you rather it be placed?

Reply:
see above